.github/renovate.json5

```diff
@@ -95,7 +95,7 @@
       // Group all eslint-related packages with `eslint` in the same PR
       matchManagers: ['npm'],
       matchPackageNames: ['eslint'],
-      matchPackagePrefixes: ['eslint-', '@typescript-eslint/'],
+      matchPackagePrefixes: ['eslint-', 'typescript-eslint'],
```

Comment:
This should be in `matchPackageNames`, as its no longer a prefix?

Reply:
```suggestion
      matchPackagePrefixes: ['eslint-'],
```